src/main/java/com/softserve/teamproject/controller/GroupController.java

```diff
@@ -79,7 +79,7 @@ public void deleteGroup(@PathVariable Integer id, Principal principal) {
    * @return groups info
    */
   @RequestMapping(value = "/groups/filter", method = RequestMethod.POST)
-  public Iterable getGroupsByFilter(@RequestBody GroupsFilterRequestWrapper requestWrapper) {
+  public Iterable getGroupsByFilter(@RequestBody GroupsFilter requestWrapper) {
```

Comment:
I'd suggest renaming variables here and in the other occurrences from wrapper to something filter-related